### DISCUSSION TITLE
Shutdown label

### DISCUSSION THREAD

**Discussion by syedhadi98** [Open-Shell/Open-Shell-Menu#1709] [Q&A]
How to change the Shutdown label on Windows 7 Style Start Menu? Because mine is be like this
![Screenshot (148)](https://github.com/Open-Shell/Open-Shell-Menu/assets/145583606/aa31bb3f-fcf3-4c61-8f60-62da4fc6e4be)

**Comment by ge0rdi** [comment]
What language do you use? (`Open-Shell Settings` -> `Language`)

The string is loaded from `StartMenuL10N.ini` file located in `Open-Shell` install dir.
Look for section with your _language_ and item named `Menu.Shutdown`.

**Comment by bonzibudd** [comment]
I think you have previously used a skin with a custom language file to change the size of the shutdown button (there is a windows Vista skin that does this). You can change it in the `StartMenuL10N.ini` file like Ge0rdi said.
